registration/pom.xml

```diff
@@ -107,6 +107,7 @@
 		<jackson.mapper.asl.version>1.7.1</jackson.mapper.asl.version>
 
 		<mosip.core.kernel.version>1.2.0.1-B1</mosip.core.kernel.version>
+		<mosip.biometric.util.version>1.2.0.1-B1-SNAPSHOT</mosip.biometric.util.version>

```

Comment:
Did you check with the latest released 1.2.0 version of the biometrics-util library?

Reply:
Yes Anusha, 1.2.0 version of biometrics-util doesnt have the class **FingerISOStandardsValidator.java**  and it is throwing the same error.